src/options.cpp

```diff
@@ -747,7 +747,7 @@ void register_options(void)
                   "Whether to indent the stuff after a leading class initializer colon");
    unc_add_option("indent_ctor_init_leading", UO_indent_ctor_init_leading, AT_UNUM,
                   "Virtual indent from the ':' for member initializers. Default=2");
-   unc_add_option("indent_ctor_init", UO_indent_ctor_init, AT_UNUM,
+   unc_add_option("indent_ctor_init", UO_indent_ctor_init, AT_NUM,
                   "Additional indenting for constructor initializer list");
```

Comment:
There should be an indication in the description message that now it accepts negative numbers and their meaning.